CONTRIBUTING.md

```diff
@@ -1,8 +1,8 @@
-# How can I contribute?
+# How can you contribute?
 
-- Add your personal details in this repository so that it appears on the [members](https://members.realdevsquad.com/) page.
+- Add your personal details in this repository by signing in on our [website](https://www.realdevsquad.com) so that it appears on the [members](https://members.realdevsquad.com/) page.
```

Comment:
Umm... When a user signs in on this website, only the user's `github_id` and `github_display_name` (if any) are stored in the database.
For users to be able to fill their personal details (first name, last name, username, company etc), we have got https://my.realdevsquad.com/signup
This form is the place where users can fill in their personal details which are required to be displayed on the members page

First the users will sign in via github on realdevsquad.com and then they should go to https://my.realdevsquad.com/signup to fill their profile